src/config/dynamic-theme-fixes.config

```diff
@@ -29590,6 +29590,37 @@ CSS
 
 ================================
 
+swiggy.com
+
+INVERT
+.logo
+.brand-logo
+.site-logo
+.main-logo
+.header-logo
+header .logo img
+nav .logo img
+.navbar .logo img
+
+NO INVERT
```

Comment:
`NO INVERT` is used for Filter mode, a different config file. Did you test this fix?